config/app_local.php.template

```diff
@@ -259,6 +259,8 @@ return [
             // Interlingua
             'ia' => ['Interlingua'], 'ina' => 'ia',
             // Italian
+            'is' => ['íslenska'], 'isl' => 'is',
```

Comment:
Would it be okay to change `íslenska` to `Íslenska` or is there a rule in Icelandic that language names should never have a capital letter? If there's no such rule, it would be more consistent to capitalize, as all the other language names start with a capital letter.

Reply:
Languages are lowercase in Icelandic, but it's the same in Russian and that has it uppercased, so I'd follow that:
`'ru' => ['Русский'], 'rus' => 'ru',`

Reply:
Okay, thanks for the confirmation, @Trendyne 🙂 

@RyckRichards Please add the capital letter.

Reply:
For your information, there is no need to add `'isl' => 'is'` because this only helps keeping old URLs by redirecting `isl` to `is`. Since Icelandic `isl` has never been published on tatoeba.org, there shouldn’t be any `isl` URL in the wild. Keeping `'isl' => 'is'` won’t hurt nevertheless.

Reply:
I think it's better to be consistent and languages are capitalised when they're the start of a sentence like any other word anyway, which is probably what you want here